src/rooms/entities/mappers/rooms.mapper.ts

```diff
@@ -0,0 +1,27 @@
+import { Room as PrismaRoom } from '@prisma/client';
+import { Room } from '../rooms.entity';
+
+export class RoomMapper {
+  static toEntity(prismaRoom: PrismaRoom): Room {
+    return Room.load(
+      prismaRoom.id,
+      prismaRoom.name,
+      prismaRoom.mode,
+      prismaRoom.maxPlayers,
+      prismaRoom.currentPlayers,
+      prismaRoom.isPrivate,
+      prismaRoom.hashedPassword,
+      prismaRoom.status,
+    );
+  }
+
+  static toPersistence(room: Room) {
```

Comment:
toPersistence 메서드에도 응답타입을 지정하면 DB 스키마가 변경됐을 때 이점이 있을거같아요

```suggestion
  static toPersistence(room: Room): PrismaRoom {
```

Reply:
좋은 것 같습니다!